model/project_ref.go

```diff
@@ -297,26 +301,36 @@ func (projectRef *ProjectRef) checkDefaultLogger() {
 	}
 }
 
-// FindOneProjectRef gets a project ref given the owner name, the repo
-// name and the project name
-func FindOneProjectRef(identifier string) (*ProjectRef, error) {
+func findOne(query db.Q) (*ProjectRef, error) {
```

Comment:
I'm kind of confused about why this is necessary-- this is the projectRef package in model so I would think findOne is pretty clear. I've also changed this to be private because there is a FindOneProjectRef, I just figured this allowed for less repeated code.

Reply:
This is the general model package. I'm talking about the scenario where you're writing code within this package and want to do a findOne of something, it's not clear what this function finds unless you look more into it. Also there's already a public FindOne which returns a repo ref (which should also be named differently, but alas)